themes/classic/templates/catalog/_partials/product-details.tpl

```diff
@@ -51,13 +51,13 @@
   {/block}
 
   {block name='product_features'}
-    {if $product.features}
+    {if $product.grouped_features}
       <section class="product-features">
         <h3 class="h6">{l s='Data sheet' d='Shop.Theme.Catalog'}</h3>
         <dl class="data-sheet">
-          {foreach from=$product.features item=feature}
+          {foreach from=$product.grouped_features item=feature}
             <dt class="name">{$feature.name}</dt>
-            <dd class="value">{$feature.value}</dd>
+            <dd class="value">{$feature.value|nl2br nofilter}</dd>
```

Comment:
I think you should escape first and then apply nl2br, otherwise this will be open to code injection attacks